src/common/utils/kaltura-params.js

```diff
@@ -142,4 +142,31 @@ function addKalturaParams(player: Player, playerConfig: PartialKPOptionsObject):
   });
 }
 
-export {addKalturaParams, handleSessionId, updateSessionIdInUrl, getReferrer, addReferrer, addClientTag};
+/**
+ * set stream priority according to playerConfig
+ * @param {Player} player - player
+ * @param {PartialKPOptionsObject} playerConfig - player config
+ * @return {void}
+ */
+function maybeSetStreamPriority(player: Player, playerConfig: PartialKPOptionsObject): void {
```

Comment:
move to `kaltura-player.js`

Reply:
I moved it to setup helpers :-) please approve

Reply:
still in `kaltura-params.js` :)